docs/en/styling-and-theming/supplemental.md

```diff
@@ -178,138 +160,36 @@ As styles in a Dojo application are mostly scoped to individual widgets, there i
-							{
-								onclick: () => {
-									updateTheme(darkTheme);
-								}
-							},
-							['dark']
-						)
-					]);
-				}
-			}),
-			v('div', { classes: this.theme(css.container) })
-		]);
-	}
-}
-```
-
 ## Making themeable widgets
 
-There are three requirements for widgets to be considered themeable:
+There are two requirements for widgets to be considered themeable:
 
-1.  The widget's class should have the `@theme()` decorator applied, passing in the widget's imported CSS module as a decorator argument.
-    -   Note that widgets can import and reference multiple CSS modules, in which case additional `@theme()` decorators can be stacked, one for each corresponding CSS import intended to be themeable.
-2.  The widget's parent class should extend `ThemedMixin`.
-3.  One or more of the widget's styling classes should be wrapped in a call to `this.theme()` when rendering the widget.
+1.  The widget's factory should have the `theme` middleware injected, `const factory = create({ theme })`
+2.  One or more of the widget's styling classes should be passed using the result from the `theme.classes(css)` call when rendering the widget.
 
-By convention, there is a fourth requirement that is useful when developing widgets intended for distribution (this is a convention that widgets in Dojo's widget library follow):
+By convention, there is a third requirement that is useful when developing widgets intended for distribution (this is a convention that widgets in Dojo's widget library follow):
 
-4.  The widget's root VDOM node - that is, the outer-most node rendered by the widget - should include a styling class named `root`. Doing so provides a predictable way to target the top-level node of a third-party themeable widget when overriding its styles in a custom theme.
+3.  The widget's root VDOM node - that is, the outer-most node rendered by the widget - should include a styling class named `root`. Doing so provides a predictable way to target the top-level node of a third-party themeable widget when overriding its styles in a custom theme.
 
-`ThemedMixin` and the `@theme()` decorator can be imported from the `@dojo/framework/core/mixins/Themed` module. Using `ThemedMixin` on a widget provides access to the `this.theme()` function.
+The `theme` middleware is imported from the `@dojo/framework/core/middleware/theme` module.
 
-### `ThemedMixin` `this.theme()` method
+### `theme.classes` method
 
 ```ts
-type SupportedClassName = string | null | undefined | boolean;
+type ClassNames = {
+	[key: string]: string;
+};
 ```
 
--   `this.theme(classes: SupportedClassName): SupportedClassName`
--   `this.theme(classes: SupportedClassName[]): SupportedClassName[]`
+-   `theme.classes<T extends ClassNames>(css: T): T`
 
-    -   Widgets can pass in one or multiple CSS class names and will receive back the corresponding amount of themed class names, suitable for passing on to the `classes` property when rendering the widget's WNodes or VNodes.
```

Comment:
This method is missing a description with this sentence gone - was that the intention?

Reply:
Added back in sentence